schema_based_on_diagram.sql

```diff
@@ -0,0 +1,56 @@
+/*CReat new databse */
+
+CREATE DATABASE clinic;
+
+CREATE TABLE patients (
+  id INT GENERATED BY DEFAULT AS IDENTITY ,
+  name VARCHAR,
+  date_of_birth DATE,
+  PRIMARY key (id)
+);
+
+CREATE TABLE medical_histories (
+  id INT GENERATED BY DEFAULT AS IDENTITY,
+  admitted_at TIMESTAMP,
+  patient_id INT REFERENCES patients (id),
+  status VARCHAR,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE invoices (
+  id INT GENERATED BY DEFAULT AS IDENTITY?
+  total_amount DECIMAL,
+  generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_histories_id REFERENCES medical_histories(id),
```

Comment:
- [ ] you should specify the Type of the key it should be Int so please make sure to add `INT` before `REFERENCES`